lib/documents/schemas/utaac_decisions.json

```diff
@@ -17,7 +17,7 @@
   ],
   "subscription_list_title_prefix": "Upper Tribunal Administrative Appeals Chamber decisions",
   "pre_production": false,
-  "summary": "<p>Check the Courts and Tribunals Judiciary website for <a href=\"http://administrativeappeals.decisions.tribunals.gov.uk/Aspx/default.aspx\">decisions made before January 2016</a>.</p>",
+  "summary": "<p>Find decisions on appeals to the Upper Tribunal (Administrative Appeals Chamber), including social security and child support appeals.</p><p>Includes decisions after December 2015. Find details of <a rel=\"external\" href=\"http://administrativeappeals.decisions.tribunals.gov.uk/Aspx/default.aspx\">older decisions.</a></p>",
```

Comment:
Jon Sanger had a look at this and suggested that the second `<p>` might read a little better and be more accessible if we tweaked it to:

`<p>This includes decisions made from January 2016 onwards. You can find details of <a rel=\"external\" href=\"http://administrativeappeals.decisions.tribunals.gov.uk/Aspx/default.aspx\">decisions made in 2015 or earlier</a> on the Courts and Tribunals Judiciary website.</p>`